craftcms/ckeditor
# allow trashed entries to be rendered when previewing

### Description
Allow trashed entries to be rendered when previewing. 

Note: #222 is needed to see the nested entries in revisions.

Testing example:
- have an entry type that contains a CKE field with the ability to nest entries in it (with revisions turned on for the section)
- create a template for this section and ensure your CKE content is showing as expected (including rendering entries nested in the CKE field) 
- create an entry, add a nested entry to the CKE field & save
- add a second nested entry to the CKE field & save
- remove the first nested entry from the CKE field & save
- go to the latest revision (revision 2 if you’re starting with a new entry) and see that you can see both nested entries showing in the revision
- click “View” when in that revision and note that without this change, you can only see the second nested entry, despite the revision having both first and second in it

### Related issues
